src/components/presentational/Section/Section.tsx

```diff
@@ -4,6 +4,7 @@ import "./Section.css"
 export interface SectionProps {
 	children?: React.ReactNode;
 	className?: string;
+	noTopMargin?: boolean;
```

Comment:
For general discussion, I've noticed a few of these "styling shortcuts" throughout this PR.  How far down that path do we want to go, or what criteria do we want to use to determine when that is appropriate to add one as opposed to having users of the widgets supply a style class?

Reply:
My 2c is zero downstream CSS overrides is the end goal we should be moving towards as this becomes more mature.  Overridding CSS scales poorly as you move forward because you either have to 1) maintain CSS classes and treat them as a "contract", bumping major versions as you change them or 2) say "override CSS at your own risk" (which is the current position of this library)

So I have been moving all the customizations / tweaks I have been making back into here; I think there are patterns we can follow like having "variants" for major different versions / styles of a component or properties like this that may be commonly used.  

(At some point we probably need to stop and think comprehensively about theming more broadly as well; there are some patterns established by mantine for instance that we could look at adopting)